packages/react-core/src/demos/DescriptionList/examples/DescriptionListBasic.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  Button,
+  DescriptionList,
+  DescriptionListTerm,
+  DescriptionListGroup,
+  DescriptionListDescription,
+  PageSection,
+  PageSectionVariants,
+  Card,
+  CardBody
+} from '@patternfly/react-core';
+import PlusCircleIcon from '@patternfly/react-icons/dist/esm/icons/plus-circle-icon';
+import { DashboardWrapper } from '@patternfly/react-core/dist/js/demos/DashboardWrapper';
+
+export const DescriptionListBasic: React.FunctionComponent = () => (
+  <DashboardWrapper>
+    <PageSection variant={PageSectionVariants.light}>
```

Comment:
If we want this demo to be as 1:1 to the Core demo as possible, then this PageSection shouldn't have the variant passed in. There should eb another PageSection before this that contains the "Main title" heading and accompanying additional text "This is a full page demo". The PageSection that contains this "Main title" should have the light variant passed in.